Octokit.Tests.Integration/Clients/CommitsClientTests.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Threading.Tasks;
 using Octokit;
+using Octokit.Models.Response;
```

Comment:
As mentioned elsewhere, this should be unnecessary after renaming the namespace...